tasks/EnvironmentConfiguration/task/task.json

```diff
@@ -0,0 +1,92 @@
+{
+  "id": "7c21ef57-08a3-4dab-a647-1c790f76ad19",
+  "name": "GenerateEnvironmentConfiguration",
+  "friendlyName": "Generate Configuration",
+  "description": "Generate, validate and inject custom application configuration in to table storage.",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Craig Gumbley",
```

Comment:
This should probably match the publisher account

Reply:
Author? Hasn’t changed since v1 but agree.